app-localize-behavior.html

```diff
@@ -164,12 +164,94 @@
     },
 
     /**
+     * If true, the global language attribute on the html tag
+     * will be used as the default language value.
+     * Use document.documentElement.lang = "de" to change the language 
+     * of the entire app.
+     */
+    useHtmlLanguage: {
```

Comment:
Instead, can you just check whether this element has a `lang` attribute, where it needs to be used? `if (this.language || this.getAttribute('lang') || html.getAttribute('lang')` or something. That way it's less user setup.